Content.Server/Atmos/Miasma/MiasmaSystem.cs

```diff
@@ -197,6 +199,11 @@ private void OnExamined(EntityUid uid, PerishableComponent component, ExaminedEv
             args.PushMarkup(Loc.GetString(description));
         }
 
+        private void OnRejuvenate(EntityUid uid, RottingComponent component, RejuvenateEvent args)
+        {
+            EntityManager.RemoveComponent<RottingComponent>(uid);
+        }
```

Comment:
I think there are issues with removing components from entities while an event subscription gets handled? E.g., see #11095.  So maybe this PR will require changes to be made to `RemCompDeferred()` first?

Reply:
Using `RemoveComponentDeferred` now.